src/libstd/io/mod.rs

```diff
@@ -290,6 +290,8 @@ pub use self::util::{copy, sink, Sink, empty, Empty, repeat, Repeat};
 pub use self::stdio::{stdin, stdout, stderr, _print, Stdin, Stdout, Stderr};
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use self::stdio::{StdoutLock, StderrLock, StdinLock};
+#[unstable(feature = "eprint", issue="40528")]
```

Comment:
Well strictly speaking the import for `_print` shouldn't be marked stable but I don't think stability attributes on imports have any effect anyway so I guess it doesn't really matter.